README.md

```diff
@@ -0,0 +1,271 @@
+# *hgvs* - manipulate biological sequence variants according to Human Genome Variation Society recommendations
+
+**Important:** biocommons packages require Python 3.6+.
+[More](https://groups.google.com/forum/#!topic/hgvs-discuss/iLUzjzoD-28)
+
+The *hgvs* package provides a Python library to parse, format, validate,
+normalize, and map sequence variants according to [Variation
+Nomenclature](http://varnomen.hgvs.org/) (aka Human Genome Variation
+Society) recommendations.
+
+Specifically, the hgvs package focuses on the subset of the HGVS
+recommendations that precisely describe sequence-level variation
+relevant to the application of high-throughput sequencing to clinical
+diagnostics. The package does not attempt to cover the full scope of
+HGVS recommendations. Please refer to
+[issues](https://github.com/biocommons/hgvs/issues) for limitations.
+
+### **Information**
```

Comment:
You don't need to bold the headings. You should also fix the header increment for the others

```suggestion
## Information
```